components/stripe/actions/capture-payment-intent/capture-payment-intent.js

```diff
@@ -0,0 +1,45 @@
+const pick = require("lodash.pick");
+const stripe = require("../../stripe.app.js");
+
+module.exports = {
+  key: "stripe-capture-payment-intent",
+  name: "Capture a Payment Intent",
+  version: "0.0.1",
+  description: "Capture the funds of an existing uncaptured payment intent.",
```

Comment:
Very small nit, but I typically don't include a period at the end of descriptions that aren't full sentences